gluten-core/src/main/scala/io/glutenproject/execution/JoinUtils.scala

```diff
@@ -104,14 +104,14 @@ object JoinUtils {
       )
 
       // Compute index for join keys in join outputs.
-      val offset = joinOutput.size - inputNodeOutput.size + selectOrigins.size
```

Comment:
Since `inputNodeOutput` has the same size with `selectOrigins`, we can simplify here.